formstest/src/main/java/org/odk/collect/formstest/InstancesRepositoryTest.java

```diff
@@ -281,6 +282,19 @@ public void softDelete_deletesInstanceDir() {
         assertThat(instanceDir.exists(), is(false));
     }
 
+    @Test
+    public void softDelete_clearsGeometryData() {
+        InstancesRepository instancesRepository = buildSubject();
+        Instance instance = instancesRepository.save(InstanceUtils.buildInstance("formid", "1", getInstancesDir())
+                .geometry("blah")
+                .geometryType("blah")
+                .build());
+
+        instancesRepository.softDelete(instance.getDbId());
```

Comment:
Maybe this language got introduced when pulling things out of the `InstancesProvider`?

Reply:
Yeah. To me, it felt analogous to what we do with forms, but I can see the argument for that not being correct.

Reply:
"Soft delete" to me means something very specific which implies that the thing that was soft deleted can be recovered. That's not the case here which is why the language feels pretty misleading to me, does that make sense?

Sorry, I commented as I was thinking so it's a little disjoint. Something like `deleteWithLogging` seems like it captures what we want.